evolution-chain/precompile/laos-evolution/contracts/LaosEvolution.sol

```diff
@@ -11,6 +11,20 @@ interface LaosEvolution {
     /// @param owner the owner of the newly created collection
     event NewCollection(uint64 collectionId, address indexed owner);
 
+    /// @notice Emitted when `tokenId` token is transferred from `from` to `to`.
+    /// @notice When the token is minted, the `from` address is 0x0 and
+    /// @notice when the token is burned, the `to` address is 0x0
+    /// @param from the previous owner of the token
+    /// @param to the new owner of the token
+    /// @param collectionId the id of the collection
+    /// @param tokenId the id of the newly minted token
+    event Transfer(
```

Comment:
since the event will not match the standard Transfer signature, I suggest to include the same params it had in the substrate side. The "slot" param is missing.

Reply:
And also, since the signature will be different, feel free to use exactly the same pattern as in the substrate side. Instead of Transfer, call it MintedWithExternalURI, and remove the "from" param.